src/features/comicReader/components/DisabledPrev.tsx

```diff
@@ -0,0 +1,11 @@
+import { IoIosSkipBackward } from "react-icons/io";
+
+const DiabledPrev = () => {
+  return (
+      <button disabled className="btn w-16 h-16  btn-circletext-3xl">
```

Comment:
Just check the spacing here, just a typo probably.

`btn w-16 h-16 btn-circle text-3xl`